packages/venia-concept/src/components/Checkout/flow.js

```diff
@@ -47,16 +46,15 @@ class Flow extends Component {
         isPaymentMethodReady: bool,
         isShippingInformationReady: bool,
         isShippingMethodReady: bool,
-        paymentMethod: string,
-        paymentTitle: string,
+        paymentDisplayPrimary: string,
```

Comment:
The `paymentMethod` was used to seed the payment method dropdown with the current selection when it was re-opened.

Since that is no longer a thing, we no longer need `paymentMethod`.